experimenter/typings/django_filters/utils.pyi

```diff
@@ -49,7 +49,7 @@ def get_model_field(model, field_name):  # -> None:
 
     ex::
 
-        f = get_model_field(Book, 'author__first_name')
```

Comment:
Should it be modifying these .pyi files?

Reply:
I don't see why not?